tools/NetCoreConversion/NetCoreConversion/Program.cs

```diff
@@ -0,0 +1,58 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Management.Automation;
+using NetCoreConversion.Utility;
+using static NetCoreConversion.NetCoreDefinitionGenerator;
+
+namespace NetCoreConversion
+{
+    public static class Program
+    {
+        public static void Main(string[] args)
+        {
+            const string rmPath = @"..\..\..\src\ResourceManager";
```

Comment:
This is valid from the build path?  I would suggest also allowing the user to specify the path in the command line args

Reply:
Please add a documentation comment for this method, as well.

Reply:
This isn't intended to be a reusable tool, per se. The work it did is already done.

Reply:
@MiYanni so then, why are we checking it in?

Reply:
@markcowl Where else would it go? It still has useful code/logic for PSD1 generation. If not here, I'll just put it in my branch in the private repo.